climada/hazard/tc_tracks.py

```diff
@@ -1643,11 +1643,9 @@ def from_FAST(cls, folder_name: str):
             TCTracks object with tracks data from the given directory of NetCDF files.
         """
 
-        LOGGER.info("Reading %s files.", len(get_file_names(folder_name)))
+        LOGGER.info(f"Reading {len(get_file_names(folder_name))} files.")
```

Comment:
```suggestion
        LOGGER.info("Reading %s files.", len(get_file_names(folder_name)))
```

Ahah I just learned something, as pylint is complaining,

Apparently, for logging, and solely for logging, the way you did it before was the right one: https://stackoverflow.com/questions/34619790/pylint-message-logging-format-interpolation

Sorry for that